django-transparent/django.dockerfile

```diff
@@ -1,7 +1,4 @@
-FROM alpine:3.10
```

Comment:
Hm, there was some explanation about newest alpine version and why it was not working with alpine:3.15.0. Have you checked if it is actually working demo with such base image version?

Reply:
it has worked, I checked. But anyway, updated hash commit for django project to version with newer psycopg2 version that fixes issues related to python3.8